edm4hep.yaml

```
@@ -471,4 +471,13 @@ datatypes :
       - float weight                // weight of this association
     OneToOneRelations:
      - edm4hep::Track   rec         // reference to the track
-     - edm4hep::MCParticle sim      // reference to the Monte-Carlo particle
\ No newline at end of file
+     - edm4hep::MCParticle sim      // reference to the Monte-Carlo particle
+
+  edm4hep::RecoParticleVertexAssociation:
+    Description: "Association between a Reconstructed Particle and a Vertex"
+    Author : "Placido Fernandez Declara"
+    Members:
+      - float weight                      // weight of this association
+    OneToOneRelations:
+     - edm4hep::ReconstructedParticle rec // reference to the track
+     - edm4hep::Vertex sim                // reference to the Monte-Carlo particle
```

```suggestion
     - edm4hep::Vertex sim                // reference to the vertex
```
Is `sim` needed as the name? I guess it is to keep the syntax the same in the k4LCIOReader converter? 
Because `sim` does not make sense in this case